packages/contracts-bedrock/scripts/deploy/DeployMIPS.s.sol

```diff
@@ -23,7 +23,7 @@ contract DeployMIPSInput is BaseDeployIO {
 
     function set(bytes4 _sel, uint256 _value) public {
         if (_sel == this.mipsVersion.selector) {
-            require(_value == 6, "DeployMIPS: unknown mips version");
+            require(_value == 7, "DeployMIPS: unknown mips version");
```

Comment:
Could these `7`'s be constants instead?

Reply:
Reorganized the constants so we can use them across tests and deploy scripts.